hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMRatisSnapshots.java

```diff
@@ -227,6 +233,20 @@ public void testInstallSnapshot() throws Exception {
         getDefaultBucketLayout()).get(followerOMMetaMngr.getOzoneKey(
         volumeName, bucketName, newKeys.get(0))));
      */
+
+    // read back data from the OM snapshot
+    OmKeyArgs omKeyArgs = new OmKeyArgs.Builder()
+        .setVolumeName(volumeName)
+            .setBucketName(bucketName)
+                .setKeyName(".snapshot/snap1/" + keys.get(0)).build();
+    OmKeyInfo omKeyInfo = null;
+    try {
+      omKeyInfo = followerOM.lookupKey(omKeyArgs);
+    } catch (Exception e) {
```

Comment:
I feel this is unnecessary to catch exception here.

Reply:
removed catch